### PR TITLE
Update dropbox.md

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ X] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/saas-cloud/dropbox.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ This section has instructions for installing the Dropbox App for Sumo Logic and
     1. **Data Source.** Select either of these options for the data source. 
       * Choose **Source Category**, and select a source category from the list. 
       * Choose **Enter a Custom Data Filter**, and enter a custom source category beginning with an underscore. Example: `_sourceCategory=MyCategory`.
```
Comment (kimsauce) [inline docs/integrations/saas-cloud/dropbox.md:129]: If you indent these lines, it'll fix the roman numeral steps

```suggestion
       * Choose **Source Category**, and select a source category from the list. 
       * Choose **Enter a Custom Data Filter**, and enter a custom source category beginning with an underscore. Example: `_sourceCategory=MyCategory`. 
```

**Review by kimsauce — DISMISSED** [review]
One suggestion, and I'd also shrink down the logo at the top to about 50px

File: docs/integrations/saas-cloud/dropbox.md (hunk 1 above)
Comment (JainM6) [reply docs/integrations/saas-cloud/dropbox.md]: indent right?

**Comment by JainM6** [comment]
> One suggestion, and I'd also shrink down the logo at the top to about 50px

Fixed. Thank you.

**Comment by kimsauce** [comment]
> > One suggestion, and I'd also shrink down the logo at the top to about 50px
> 
> Fixed. Thank you.

Hi @JainM6 , the icon is still showing as the same to me. I've fixed it here and replaced with a high-res version - can you please take a look? https://github.com/SumoLogic/sumologic-documentation/pull/1759

**Pull request merged**